torch/quantization/fx/quantize.py

```diff
@@ -491,6 +495,52 @@ def in_nodes(a: Argument, nodes: Set[Node]) -> bool:
 
     return result_graph
 
+def handle_cat_nodes(
+        model: torch.nn.Module, observed_graph: Graph, matches: Dict[str, MatchResult],
+        quants: Dict[str, List[Tuple[DefaultQuantizeHandler, Callable]]],
+        activation_post_process_map: Dict[str, List[str]],
+        modules: Dict[str, torch.nn.Module]):
+    observed_nodes: Set[Node] = set()
+    # activation_post_process for cat nodes
+    app_for_cat_nodes: Dict[Node, torch.nn.Module] = dict()
```

Comment:
optional: maybe `actpp` instead of `app`?  `app` means so many things :)

Reply:
sure